cmp/grid/impl/ColumnWidthCalculator.js

```diff
@@ -87,26 +84,38 @@ export class ColumnWidthCalculator {
         } catch (e) {
             console.warn(`Error calculating max data width for column "${column.colId}".`, e);
         } finally {
-            this.setCellElActive(false);
+            this.resetClassNames();
```

Comment:
think we can remove this -- we are doing a reset already before every measurement.

Reply:
This removes the 'xh-grid-autosize-cell--active' class, which hides the cell. Think we always want to do that after an autosize, especially if it throws at any point.